site/en/docs/webstore/group-publishers/index.md

```diff
@@ -1,115 +1,125 @@
+
+These steps create a new Group Publisher account. The Google Group you selected is linked to this new publisher account, and the group email is the new group publisher account's email.
+
+## Adding developers to the group publisher
+
+Your dashboard page will show the new Group Publisher account and the linked
+Google Group. You can [add or remove developers](https://groups.google.com/).
+
+**Any Chrome Web Store developer who is a member of the linked group can act on behalf of the new publisher account**. For example, they can edit items, publish items, and edit the publisher's display name.
+
+## Publishing using a group publisher
+
+In the top right-hand corner of the [Chrome Web Store developer console](https://chrome.google.com/webstore/devconsole) is a pull-down that contains the following items:
+
+* Your developer user name (your individual publisher)
+* Any group publishers that you are a member of.
+
+Select the publisher you want to use. This displays the items already uploaded for that publisher. Any new items that you upload are associated with the selected publisher.
 
 ## Things to note
 
-- Each member of the group must pay the developer fee.
-- You can only set up group publishing once and create one group publisher account.
-- You must be the owner or administrator of a Google Group to link the group. The drop-down list on
-  the dashboard page only shows groups that you're a direct member of (for example, if you are a
-  member of Group A and Group A is a member of Group B, you are not a direct member of Group B, and
-  Group B is not shown on the drop-down list).
-- Once you finish the group publishing setup, it can take up to 30 minutes for developers in your
-  group to see the changes.
-- Your domain is the default domain of the new group publisher account. A group publisher can
-  publish items either to the public or to its default domain. A publisher cannot publish to other
-  domains.
-- A developer can act on behalf of multiple publishers if the developer is a member of multiple
-  Google Groups linked to publishers.
-- If a merchant account is needed or is already set up, any member of the group can choose to
-  re-link their own merchant accounts to the publisher on the dashboard page.
-- When you want to create new store items, you will have the opportunity to choose whether to make
-  them part of your personal publisher account or the group publisher account before you upload the
-  item.
+* You can only set up group publishing once and create one group publisher account.
+* You must be the owner or manager of a Google Group to link the group. The drop-down list on the dashboard page only shows groups that you're a direct member of (for example, if you are a member of Group A and Group A is a member of Group B, you are not a direct member of Group B, and Group B is not shown on the drop-down list).
```

Comment:
we should make it clear that this description is not only true for the display, but also true for membership checks - indirect members are not considered a member of the publisher group even though they are considered  part of the google group.

Reply:
Done